### PR TITLE
Adding dependency loading mechansim for NetCore

### PR DESCRIPTION
**Pull request by markcowl**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Profile/Commands.Profile/Commands.Profile.Netcore.csproj [hunk 1]
```diff
@@ -36,9 +36,17 @@
   <ItemGroup>
     <PreLoadAssemblies Include="$(ProjectDir)..\..\..\lib\Newtonsoft.Json.9.dll"/>
     <PreLoadAssemblies Include="$(ProjectDir)..\..\..\lib\Newtonsoft.Json.10.dll"/>
+    <PreLoadAssemblies Include="$(ProjectDir)..\..\..\lib\Microsoft.IdentityModel.Clients.ActiveDirectory.dll"/>
```
Comment (cormacpayne) [inline src/ResourceManager/Profile/Commands.Profile/Commands.Profile.Netcore.csproj]: @markcowl discussed offline about changing the path to be

`lib\Microsoft.IdentityModel.Clients.ActiveDirectory\{{version}}\Microsoft.IdentityModel.Clients.ActiveDirectory.dll`

**Comment by markcowl** [comment]
netsign: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/206/

**Comment by markcowl** [comment]
netsign: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/207/

**Comment by markcowl** [comment]
netsign: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/214/

**Pull request merged**